docs/announcements/github-release-v0.1.79.md

```
@@ -0,0 +1,20 @@
+# 🚀 Release Highlights — v0.1.79 (2026-04-20)
+
+### ⚡ [Better Fast Mode Options](https://github.com/massgen/MassGen/blob/main/massgen/configs/features/fast_iteration.yaml)
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)